src/overlays/actors/ovl_En_Insect/z_en_insect.h

```diff
@@ -9,6 +9,16 @@ struct EnInsect;
 
 typedef void (*EnInsectActionFunc)(struct EnInsect*, GlobalContext*);
 
+#define INSECT_FLAG_TEMP_AND_ALIVE (1 << 0) // Set when the bug is temporary and is not drowning.
+#define INSECT_FLAG_ON_GROUND (1 << 1) // Set when the bug was dropped and is standing on the ground.
+#define INSECT_FLAG_TEMP (1 << 2) // Set when the bug disappears after a while.
+#define INSECT_FLAG_UNCATCHABLE (1 << 3) // Set when the bug is about to disappear and can no longer be caught.
+#define INSECT_FLAG_FOUND_SOIL (1 << 4)
+#define INSECT_FLAG_SOIL_CLOSE (1 << 5)
```

Comment:
I'm hesitant if this insect flag should be named by usage (the current `INSECT_FLAG_SOIL_CLOSE`) or by effect (disabling OC collision)

I guess naming by usage makes the intent clearer so it's probably better as is